grpc/rust/compile/compile.rs

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
We now pass in all the `tonic_build` configuration via environment variables, which are set by the `rust_tonic_compile` rule.